Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml

```diff
@@ -152,8 +152,8 @@
       entries:
       - id: CMPouchAutoinjector
         amount: 1
-      #- id: CMPouchMedkit
-      #  amount: 2
+      - id: CMPouchMedkit
+        amount: 2
```

Comment:
Ok in retrospect I didn't add these in yet on purpose, since we don't have it implemented where it requires medical skill to use effectively. You can comment all uses of CMPouchMedkit or wait until I implement that to merge this.

Reply:
done.